exercises/csharp/TriviaGame/TriviaGame/Game.cs

```diff
@@ -1,81 +1,84 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Text;
 
 namespace TriviaGame
 {
     public class Game
     {
-        List<string> players = new List<string>();
+        private readonly List<string> players = new List<string>();
 
-        int[] places = new int[6];
-        int[] purses = new int[6];
+        private readonly int[] places = new int[Constants.MaxPlayers];
+        private readonly int[] purses = new int[Constants.MaxPlayers];
 
-        bool[] inPenaltyBox = new bool[6];
+        private readonly LinkedList<string> popularQuestions = new LinkedList<string>();
+        private readonly LinkedList<string> scienceQuestions = new LinkedList<string>();
+        private readonly LinkedList<string> sportsQuestions = new LinkedList<string>();
+        private readonly LinkedList<string> rockQuestions = new LinkedList<string>();
 
-        LinkedList<string> popQuestions = new LinkedList<string>();
-        LinkedList<string> scienceQuestions = new LinkedList<string>();
-        LinkedList<string> sportsQuestions = new LinkedList<string>();
-        LinkedList<string> rockQuestions = new LinkedList<string>();
+        private int currentPlayer = 0;
+        private bool isGettingOutOfPenaltyBox;
 
-        int currentPlayer = 0;
-        bool isGettingOutOfPenaltyBox;
+        private readonly Board board;
 
-        public Game()
+        private enum QuestionCategory
         {
-            for (int i = 0; i < 50; i++)
-            {
-                popQuestions.AddLast("Pop Question " + i);
-                scienceQuestions.AddLast(("Science Question " + i));
-                sportsQuestions.AddLast(("Sports Question " + i));
-                rockQuestions.AddLast(CreateRockQuestion(i));
-            }
+            Pop,
+            Science,
+            Sports,
+            Rock
         }
 
-        public string CreateRockQuestion(int index)
+        public Game()
         {
-            return "Rock Question " + index;
-        }
+            board = new Board();
 
-        public bool IsPlayable()
-        {
-            return (HowManyPlayers() >= 2);
+            CreateQuestions();
         }
 
-        public bool Add(string playerName)
+        public bool HasMinimumRequiredPlayers { get => players.Count >= 2; }
+        public int NumberOfPlayers { get => players.Count; }
+
+        private void CreateQuestions()
         {
+            for (int i = 0; i < 50; i++)
```

Comment:
consider naming `int i` `int index` or something else `int index` this into the template literal `{questionNumber}` 
you could then also delete line 46